migrations/20180507162526_create_license_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+use \Phinx\Db\Adapter\MysqlAdapter;
+
+class CreateLicenseTable extends AbstractMigration
+{
+	public function up()
+	{
+		$this->table('hxl_license')
+			->addColumn('name', 'string', [
```

Comment:
Are we likely to search by name? ie. does it need an index

Reply:
>Are we likely to search by name?
no